css/class-names/index.js

```diff
@@ -0,0 +1,272 @@
+/**
+ * Getting an error here? Remember to run `npm run prepare:extension` before debugging extension.
+ */
+
+// @ts-nocheck
+const fs = require('fs-extra');
+const path = require('path');
+// const AtlasTokenTypes = require('../dist/tokens');
```

Comment:
just curious if `AtlasTokenTypes` is intended to be used in the file somewhere.

Reply:
we can remove the comment, done!